superchain/configs/chainList.json

```diff
@@ -0,0 +1,130 @@
+[
+  {
+    "name": "Base Sepolia",
+    "identifier": "sepolia/base",
+    "chainId": 84532,
+    "public_rpc": "https://sepolia.base.org",
+    "explorer": "https://sepolia-explorer.base.org",
+    "superchain_level": 2
+  },
```

Comment:
Have we considered approximately mirroring the JSON structure returned by https://chainid.network/chains.json or other similar endpoints for https://github.com/ethereum-lists/chains? For example, this would result in the following changes:
- `rpc` and `explorer` become arrays, which is nice so we don't enshrine a single provider
- Use the `parent` key to specify the appropriate L1
- All new new fields are left untouched

```json
  {
    "name": "Base Sepolia",
    "identifier": "sepolia/base",
    "chainId": 84532,
    "rpc": ["https://sepolia.base.org"],
    "explorers": ["https://sepolia-explorer.base.org"],
    "superchain_level": 2,
    "parent": { ... }
  },
```

Reply:
I tried that out here [30701d3](https://github.com/ethereum-optimism/superchain-registry/pull/265/commits/30701d3369cc2decfc839ec7e27cc630d13b388a) LMKWYT. 

My general comments are:
* we don't support multiple RPCs or explorers in the actual underlying registry structures at the moment (if we like this change, probably we should do that)
* we don't scrape any bridge URLs to put in the "parent" field at the moment
* we aren't using the chain agnostic identification scheme https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md as the ethereum-list repo does
* I hardcoded the parent type as "L2" for now.

Reply:
Sounds good. Not a blocker for me, just wanted to flag it as an option since it's possible e.g. MetaMask and others already have scripts to parse that JSON structure. 

I do think it's ok to make RPC and explorers arrays in the output though, even though it's not supported as such in the input. Will defer to you to resolve :)